v3/cypress/e2e/formula/formula-component.spec.ts

```diff
@@ -83,15 +83,16 @@ context("Formula Engine", () => {
         "❌ Undefined symbol v1"
       ])
     })
-    it("Formula in a new dataset", () => {
+    // Skipped for now until insert cases on input row is implemnted
```

Comment:
How difficult would it be to implement this? Isn't this already done  in `table.spec.ts` lines 738-744?

Reply:
Part of this PR removed the single case created when a new table is first created. 
Input row index cell does not open the index column menu so new cases cannot be inserted without having to enter data.
The `table.spec.ts` lines 738-744 tests that a new case is created at the bottom when user enters data into the input line.

Reply:
done